src/plr/data.cljs

```diff
@@ -26,4 +26,4 @@
 
 (def arrays ["APL" "J" "q" "BQN" "Julia" "Fortran" "MATLAB" "R" "Futhark" "Mathematica" "Chapel"])
 
-(def system ["C" "C++" "Rust" "Swift" "Nim" "Zig" "Go" "D" "Ada", "Pascal"])
\ No newline at end of file
+(def system ["C" "C++" "Rust" "Swift" "Nim" "Zig" "Go" "D" "Ada" "Pascal" "Odin" "Hare" "Jai" "Crystal" "Mojo"])
```

Comment:
```suggestion
(def system ["C" "C++" "Rust" "Swift" "Nim" "Zig" "Go" "D" "Ada" "Pascal" "Odin" "Hare" "Crystal" "Mojo"])
```

Reply:
Yea, lets just remove Jai. I will merge after that.

Reply:
Ah, I saw the message before the suggestion but the PR is updated.